Cargo.toml

```diff
@@ -4,14 +4,14 @@ members = [
 	"consensus",
 	"crypto",
 	"snark-keys",
+	"base/test-staged_ledger_diff",
```

Comment:
why create a new crate instead of making it part of the base crate?

Reply:
I learn from your test-serialization under the protocol folder. Cargo throws me an error suggesting that I should add this in the workspace Cargo.toml

Reply:
Thank you for pointing this out. I just fix it.

Reply:
Fix in #122